<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Input Text Box</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>951</width>
     <height>251</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Input text to be analyzed here! Default Algorithm is N-Gram</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="input">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>30</y>
     <width>691</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tree">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>AuthorProfiles</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonFolder">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Test Folder</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>99.999999000000003</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>N-Gram</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>MinFreq</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Word Profile Test?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
